bazel/gapic.bzl

```diff
@@ -26,16 +26,18 @@ def cc_gapic_library(name, service_dirs = [], googleapis_deps = []):
             this library depends on.
     """
 
-    src_dirs = service_dirs + [d + "internal/" for d in service_dirs]
+    code_glob = [d + i + f for d in service_dirs for i in ["", "internal/"] for f in ["*.h", "*.cc"]]
+
+    sources_glob = [d + "internal/*_sources.cc" for d in service_dirs] + [d + "internal/streaming.cc" for d in service_dirs]
```

Comment:
Some judicious line breaking would be useful here.

Reply:
I tried.